### ISSUE TITLE
SpotifyXP on Pentium 2 laptop

### ISSUE THREAD

**Issue by djsubtronic** [SpotifyXP/SpotifyXP#26]
I have a strange issue on a Pentium 2 laptop. Because it does not have SSE2, I can't use VLC 3.0.20 so I used the latest version that runs without SSE2 support, VLC 2.0.8. I also tried 3.0.20, because it does "run" in some way but it's just a command line (no VLC GUI).

Even with VLC 2.0.8, it has worked a couple of times. Basically I get the "SpotifyXP seems to be stuck" message, but if I just say "no", it has managed to proceed and launch the app about 3 times out of 20. When the app has launched, I was able to browse my library and play music without issues. But the other 17 or so times, it just stays stuck on the "creating api" screen and I have to force close it myself.

I can't figure out any pattern as to why it has managed to run OK a few times though, but I have attached a log from one of the times that it has got stuck and not opened.

[debug.txt](https://github.com/SpotifyXP/SpotifyXP/files/14348556/debug.txt)

**Comment by djsubtronic** [comment]
Just noticed there are nightly builds - I have tried the latest one from here: https://github.com/SpotifyXP/SpotifyXP/actions/runs/7976996138

It loaded up fine the first time I ran it but unfortunately I did not have logging enabled. Subsequently it has always got stuck at "creating api" although this version does not display the popup as 2.0.2.

Attached is a log for this version after it gets stuck on "creating api".

[debug-nightly.txt](https://github.com/SpotifyXP/SpotifyXP/files/14351460/debug-nightly.txt)
